src/main/java/htsjdk/samtools/cram/encoding/reader/DataSeriesReader.java

```
@@ -50,36 +48,26 @@ public DataSeriesReader(final DataSeriesType valueType,
                             final BitInputStream bitInputStream,
                             final Map<Integer, InputStream> inputMap) {
 
-        final EncodingFactory f = new EncodingFactory();
-        final Encoding<T> encoding = f.createEncoding(valueType, params.id);
```

`createEncoding` now throws instead of returning `null`

---

agreed